package.json

```diff
@@ -43,9 +44,6 @@
   "peerDependenciesMeta": {
     "@types/eslint": {
       "optional": true
-    },
-    "eslint-config-prettier": {
```

Comment:
It's actually required by legacy recommended config at

https://github.com/prettier/eslint-plugin-prettier/blob/17c3c109cb9022301e804e418cae38ff489f8e69/eslint-plugin-prettier.js#L82

Reply:
I can add it back if you want, but specifying a peer dependency as optional in `peerDependenciesMeta` doesn't actually do anything unless also specified in `peerDependencies`: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependenciesmeta

Should I take it back anyway?

Reply:
Sure, I'll add it to both `peerDependencies` and `peerDependenciesMeta` for now then. What version of `eslint-config-prettier` should be required from peer dependencies?

Reply:
I believe '*' is fine enough? How do you think?

Reply:
Yep, probably fine. Added it now!